docs/taxon-concept-relationship-type-vocabulary.md

```diff
@@ -254,15 +215,16 @@ do not edit the markdown directly, but make any changes in the YAML file.
 			<td>Definition</td>
 			<td><p>The taxonomic meanings of the subject and object taxon concepts intersect,  <em>i.e.</em> they have at least one member in common.</p></td>
 		</tr>
-		<tr>
-			<td>Comments</td>
-			<td><p><code>intersects</code> is the opposite of <code>isDisjointFrom</code> and the union of  <code>isCongruentWith</code>, <code>hasProperSubset</code>, <code>isProperSubsetOf</code> and  <code>partiallyOverlaps</code>, meaning it can be any of these relationships. This  relationship type can be used when the more precise nature of the  relationship is not known, for example when dealing with statements in  traditional synonymies.</p></td>
-		</tr>
 		<tr>
 			<td>Controlled value</td>
 			<td>intersects</td>
 		</tr>
 	</tbody>
 </table>
 
+
+**Comments**
+
+`intersects` is the opposite of `isDisjointFrom` and the union of  `isCongruentWith`, `hasProperSubset`, `isProperSubsetOf` and  `partiallyOverlaps`, meaning it can be any of these relationships. This  relationship type can be used when the more precise nature of the  relationship is not known, for example when dealing with statements in  traditional synonymies.
```

Comment:
```suggestion
`intersects` is the opposite of `isDisjointFrom` and is the superproperty of  `isCongruentWith`, `hasProperSubset`, `isProperSubsetOf` and  `partiallyOverlaps`, meaning it can be any of these relationships. This  relationship type can be used when the more precise nature of the  relationship is not known, for example when dealing with statements in  traditional synonymies.
```